datafeeder/pom.xml

```diff
@@ -341,6 +350,11 @@
       <artifactId>jersey-hk2</artifactId>
       <version>${jersey-version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.univocity</groupId>
+      <artifactId>univocity-parsers</artifactId>
+      <version>2.9.1</version>
+    </dependency>
```

Comment:
looks like it is not in use anymore (checking `com.univocity.*` imports in the current diff)